docker-assets/appliance-initialize.sh

```diff
@@ -1,5 +1,52 @@
-#!/bin/sh
-[[ -d /var/opt/rh/rh-postgresql95/lib/pgsql/data/base ]] && exit 0
-[[ -s /etc/default/evm ]] && source /etc/default/evm
-echo "Initializing Appliance, please wait ..."
-appliance_console_cli --region 0 --internal --password smartvm --key
+#!/bin/bash
+
+# Source EVM environment 
+[ -f /etc/default/evm ] &&  . /etc/default/evm
+[[ -s ${CONTAINER_SCRIPTS_ROOT}/container-deploy-common.sh ]] && source "${CONTAINER_SCRIPTS_ROOT}/container-deploy-common.sh"
+
+function create_v2_key() {
+  V2_KEY=$(ruby -ropenssl -rbase64 -e 'puts Base64.strict_encode64(Digest::SHA256.digest(OpenSSL::Random.random_bytes(32))[0, 32])')
+  write_v2_key
```

Comment:
Where does this function live?  If it's part of manageiq-appliance, maybe we can move this create_v2_key function there too?

Reply:
Clearly this lives in https://github.com/ManageIQ/manageiq-pods/blob/master/images/miq-app/docker-assets/container-scripts/container-deploy-common.sh#L23-L30 :trollface: 

I put this function here because the other images will never need it. The key is generated and stored in a secret by OpenShift so they don't need to know how to actually build the key value.